test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1856,7 +1856,7 @@ BOOST_AUTO_TEST_CASE(uncalled_blockhash)
 		contract C {
 			function f() public view returns (bytes32)
 			{
-				return (block.blockhash)(block.number - 1);
+				return (blockhash)(block.number - 1);
```

Comment:
The name of this test case now is very misleading, it is not about uncalled `blockhash` anymore. I think this was broken in the `var` change PR though.

Reply:
I would say the test is still fine. `blockhash` is used as a stand-alone expression without being called directly. The type checker still has to assign a type to the parenthesized expression.